src/webgpu/gpu_test.ts

```diff
@@ -167,6 +167,33 @@ export class GPUTest extends Fixture {
     });
   }
 
+  expectAlternativeContents(
```

Comment:
expectContentsMultipleValidValues?
expectContentsNonDeterministic?

Reply:
Done. I chose the first one.